### PR TITLE
[All Hosts] (manifests) reorg and update main manifest articles for unified manifest

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
This PR does the following:
1. Splits the existing manifest overview into a format-neutral overview and an overview of the XML manifest.
2. Creates a new overview for the unified manifest.
3. Renames the existing overview of the unified manifest to be a comparison of the two kinds of manifests which is what it really is.
4. Moves content from the Outlook manifest overview to the XML manifest overview if it applies to all add-ins, not just Outlook. This was actually most of the content in it.
5. Renames the Outlook manifest overview to make it format-neutral and to be specifically about manifest items that apply only to Outlook.
6. Updates, as needed, links to any of the above in other articles. 

NOTE: FOR ARTICLES THAT WERE CHANGED ONLY AS PART OF #6 ABOVE, I DID NOT CHANGE THE METADATA DATE.

### CODE REVIEW

File: docs/outlook/manifests.md [hunk 1]
```diff
@@ -1,407 +1,66 @@
 ---
-title: Outlook add-in manifests
-description: Get an overview of the two kinds of manifests available for Outlook Add-ins.
-ms.date: 05/20/2023
+title: Manifest configuration for Outlook Add-ins
+description: Get an overview of the add-in manifest markup and JSON that is relevant only to Outlook.
+ms.date: 05/24/2023
+
 ms.localizationpriority: high
```
Comment (samantharamon) [inline docs/outlook/manifests.md:6]:
```suggestion
ms.date: 05/24/2023
ms.localizationpriority: high
```

File: docs/outlook/manifests.md [hunk 2]
```diff
@@ -1,407 +1,66 @@
 ---
-title: Outlook add-in manifests
-description: Get an overview of the two kinds of manifests available for Outlook Add-ins.
-ms.date: 05/20/2023
+title: Manifest configuration for Outlook Add-ins
```
Comment (samantharamon) [inline docs/outlook/manifests.md:2]:
```suggestion
title: Manifest configuration for Outlook add-ins
```

File: docs/outlook/manifests.md [hunk 3]
```diff
@@ -1,407 +1,66 @@
...
+description: Get an overview of the add-in manifest markup and JSON that is relevant only to Outlook.
+ms.date: 05/24/2023
+
 ms.localizationpriority: high
 ---
 
-# Outlook add-in manifests
+# Manifest configuration for Outlook Add-ins
```
Comment (samantharamon) [inline docs/outlook/manifests.md:9]:
```suggestion
# Manifest configuration for Outlook add-ins
```

File: docs/outlook/manifests.md [hunk 4]
```diff
@@ -1,407 +1,66 @@
...
-
-These settings are directly related to the activation rules in the **\<Rule\>** element. For example, if an add-in specifies that it should appear on a message in compose mode, an **ItemEdit** form must be specified.
-
-For more details, please refer to the [Schema reference for Office Add-ins manifests (v1.1)](../develop/add-in-manifests.md).
 
-## App domains
+# [Unified manifest for Microsoft 365 (developer preview)](#tab/jsonmanifest)
 
 The domain of the add-in start page that you specify in the **\<SourceLocation\>** element is the default domain for the add-in. Without using the **\<AppDomains\>** and **\<AppDomain\>** elements, if your add-in attempts to navigate to another domain, the browser or webview control will open a new window outside of the add-in pane. In order to allow the add-in to navigate to another domain within the add-in pane, add an **\<AppDomains\>** element and include each additional domain in its own **\<AppDomain\>** sub-element in the add-in manifest.
```
Comment (samantharamon) [inline docs/outlook/manifests.md:35]:
```suggestion
```
Leftover from previous content?

File: docs/outlook/manifests.md [hunk 5]
```diff
@@ -413,6 +72,15 @@ Activation rules can be used to activate an add-in based on one or more of the f
 
 For details and samples of activation rules, see [Activation rules for Outlook add-ins](activation-rules.md).
 
+## Form settings
+
+> [!NOTE]
+> Form settings aren't supported in Outlook add-ins that use the unified manifest for Microsoft 365.
+
+The **\<FormSettings\>** element is used by older Outlook clients, which only support schema 1.1 and not **\<VersionOverrides\>**. Using this element, developers define how the add-in will appear in such clients. There are two parts - **ItemRead** and **ItemEdit**. **ItemRead** is used to specify how the add-in appears when the user reads messages and appointments. **ItemEdit** describes how the add-in appears while the user is composing a reply, new message, new appointment or editing an appointment where they are the organizer.
```
Comment (samantharamon) [inline docs/outlook/manifests.md:80]:
```suggestion
The **\<FormSettings\>** element is used by older Outlook clients, which only support schema 1.1 and not **\<VersionOverrides\>**. Using this element, developers define how the add-in will appear in such clients. There are two parts&mdash;**ItemRead** and **ItemEdit**. **ItemRead** is used to specify how the add-in appears when the user reads messages and appointments. **ItemEdit** describes how the add-in appears while the user is composing a reply, new message, new appointment or editing an appointment as an organizer.
```

File: docs/outlook/manifests.md (hunk 4 above)
Comment (Rick-Kirkham) [reply docs/outlook/manifests.md:35]: Strange. This paragraph isn't in the source file. I wonder why it's showing up here.

**Pull request merged**